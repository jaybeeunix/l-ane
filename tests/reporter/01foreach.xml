<?xml version='1.0' encoding='US-ASCII' standalone='yes'?>
<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>

<html xmlns='http://www.w3.org/1999/xhtml' xmlns:rpt='http://l-ane.net/l-ane-xml/xmlReporter0'>

<head>
<title>L&#x2019;&#xe2;ne XML Reporter Test: Constant foreach Tests</title>
</head>

<body>
<h1>Constant foreach Tests</h1>
<h2>Simple foreach</h2>
<rpt:foreach set='SetConstants.1 5' element='i'><p>This is iteration <rpt:output value='i.value'/>.</p>
</rpt:foreach>

<h2>Nested Simple foreach</h2>
<h3>Two Levels</h3>
<rpt:foreach set='SetConstants.1 3' element='j'>
<rpt:foreach set='SetConstants.1 4' element='k'><p>Outter loop <rpt:output value='j.value'/>, Inner loop <rpt:output value='k.value'/>.</p>
</rpt:foreach></rpt:foreach>

<h3>Three Levels</h3>
<rpt:foreach set='SetConstants.1 3' element='j'>
<rpt:foreach set='SetConstants.1 4' element='k'>
<rpt:foreach set='SetConstants.1 5' element='l'><p>Outter loop <rpt:output value='j.value'/>, Inner loop <rpt:output value='k.value'/>, Deep loop <rpt:output value='l.value'/>.</p>
</rpt:foreach></rpt:foreach></rpt:foreach>
</body>
</html>
