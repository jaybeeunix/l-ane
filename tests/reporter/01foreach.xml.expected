<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
<html xmlns='http://www.w3.org/1999/xhtml'>

<head>
<title>L’âne XML Reporter Test: Constant foreach Tests</title>
</head>

<body>
<h1>Constant foreach Tests</h1>
<h2>Simple foreach</h2>
<p>This is iteration 1.</p>
<p>This is iteration 2.</p>
<p>This is iteration 3.</p>
<p>This is iteration 4.</p>
<p>This is iteration 5.</p>


<h2>Nested Simple foreach</h2>
<h3>Two Levels</h3>

<p>Outter loop 1, Inner loop 1.</p>
<p>Outter loop 1, Inner loop 2.</p>
<p>Outter loop 1, Inner loop 3.</p>
<p>Outter loop 1, Inner loop 4.</p>

<p>Outter loop 2, Inner loop 1.</p>
<p>Outter loop 2, Inner loop 2.</p>
<p>Outter loop 2, Inner loop 3.</p>
<p>Outter loop 2, Inner loop 4.</p>

<p>Outter loop 3, Inner loop 1.</p>
<p>Outter loop 3, Inner loop 2.</p>
<p>Outter loop 3, Inner loop 3.</p>
<p>Outter loop 3, Inner loop 4.</p>


<h3>Three Levels</h3>


<p>Outter loop 1, Inner loop 1, Deep loop 1.</p>
<p>Outter loop 1, Inner loop 1, Deep loop 2.</p>
<p>Outter loop 1, Inner loop 1, Deep loop 3.</p>
<p>Outter loop 1, Inner loop 1, Deep loop 4.</p>
<p>Outter loop 1, Inner loop 1, Deep loop 5.</p>

<p>Outter loop 1, Inner loop 2, Deep loop 1.</p>
<p>Outter loop 1, Inner loop 2, Deep loop 2.</p>
<p>Outter loop 1, Inner loop 2, Deep loop 3.</p>
<p>Outter loop 1, Inner loop 2, Deep loop 4.</p>
<p>Outter loop 1, Inner loop 2, Deep loop 5.</p>

<p>Outter loop 1, Inner loop 3, Deep loop 1.</p>
<p>Outter loop 1, Inner loop 3, Deep loop 2.</p>
<p>Outter loop 1, Inner loop 3, Deep loop 3.</p>
<p>Outter loop 1, Inner loop 3, Deep loop 4.</p>
<p>Outter loop 1, Inner loop 3, Deep loop 5.</p>

<p>Outter loop 1, Inner loop 4, Deep loop 1.</p>
<p>Outter loop 1, Inner loop 4, Deep loop 2.</p>
<p>Outter loop 1, Inner loop 4, Deep loop 3.</p>
<p>Outter loop 1, Inner loop 4, Deep loop 4.</p>
<p>Outter loop 1, Inner loop 4, Deep loop 5.</p>


<p>Outter loop 2, Inner loop 1, Deep loop 1.</p>
<p>Outter loop 2, Inner loop 1, Deep loop 2.</p>
<p>Outter loop 2, Inner loop 1, Deep loop 3.</p>
<p>Outter loop 2, Inner loop 1, Deep loop 4.</p>
<p>Outter loop 2, Inner loop 1, Deep loop 5.</p>

<p>Outter loop 2, Inner loop 2, Deep loop 1.</p>
<p>Outter loop 2, Inner loop 2, Deep loop 2.</p>
<p>Outter loop 2, Inner loop 2, Deep loop 3.</p>
<p>Outter loop 2, Inner loop 2, Deep loop 4.</p>
<p>Outter loop 2, Inner loop 2, Deep loop 5.</p>

<p>Outter loop 2, Inner loop 3, Deep loop 1.</p>
<p>Outter loop 2, Inner loop 3, Deep loop 2.</p>
<p>Outter loop 2, Inner loop 3, Deep loop 3.</p>
<p>Outter loop 2, Inner loop 3, Deep loop 4.</p>
<p>Outter loop 2, Inner loop 3, Deep loop 5.</p>

<p>Outter loop 2, Inner loop 4, Deep loop 1.</p>
<p>Outter loop 2, Inner loop 4, Deep loop 2.</p>
<p>Outter loop 2, Inner loop 4, Deep loop 3.</p>
<p>Outter loop 2, Inner loop 4, Deep loop 4.</p>
<p>Outter loop 2, Inner loop 4, Deep loop 5.</p>


<p>Outter loop 3, Inner loop 1, Deep loop 1.</p>
<p>Outter loop 3, Inner loop 1, Deep loop 2.</p>
<p>Outter loop 3, Inner loop 1, Deep loop 3.</p>
<p>Outter loop 3, Inner loop 1, Deep loop 4.</p>
<p>Outter loop 3, Inner loop 1, Deep loop 5.</p>

<p>Outter loop 3, Inner loop 2, Deep loop 1.</p>
<p>Outter loop 3, Inner loop 2, Deep loop 2.</p>
<p>Outter loop 3, Inner loop 2, Deep loop 3.</p>
<p>Outter loop 3, Inner loop 2, Deep loop 4.</p>
<p>Outter loop 3, Inner loop 2, Deep loop 5.</p>

<p>Outter loop 3, Inner loop 3, Deep loop 1.</p>
<p>Outter loop 3, Inner loop 3, Deep loop 2.</p>
<p>Outter loop 3, Inner loop 3, Deep loop 3.</p>
<p>Outter loop 3, Inner loop 3, Deep loop 4.</p>
<p>Outter loop 3, Inner loop 3, Deep loop 5.</p>

<p>Outter loop 3, Inner loop 4, Deep loop 1.</p>
<p>Outter loop 3, Inner loop 4, Deep loop 2.</p>
<p>Outter loop 3, Inner loop 4, Deep loop 3.</p>
<p>Outter loop 3, Inner loop 4, Deep loop 4.</p>
<p>Outter loop 3, Inner loop 4, Deep loop 5.</p>

</body>
</html>
